dianna/methods/lime.py

```diff
@@ -276,3 +282,27 @@ def moveaxis_function(data):
         if self.preprocess_function is None:
             return moveaxis_function
         return lambda data: self.preprocess_function(moveaxis_function(data))
+
+    @staticmethod
+    def get_explanation_values(label: int, explanation: ImageExplanation) -> NDArray:
+        """Get the values from LIME in a salience map.
+
+        Leverages the `ImageExplanation` class from LIME to generate salience maps
+        based on the segmentation that was produced on the images, also stored in
+        `ImageExplanation`.
```

Comment:
Please also consider adding more explanation about the scores, e.g. "The importance values are weight of surrogate models (e.g. weight of a ridge model) which are used by LIME to evaluate the contribution of each data/pixel/segmentation to the predicted classes."